infrastructure/main.tf

```diff
@@ -1,5 +1,5 @@
 module "mwaa" {
-  source                           = "https://github.com/NASA-IMPACT/mwaa_tf_module/releases/download/v1.1.9/mwaa_tf_module.zip"
+  source                           = "https://github.com/NASA-IMPACT/mwaa_tf_module/releases/download/v1.1.10/mwaa_tf_module.zip"
```

Comment:
Do you need to add the `mcp_deployment` variable after the mwaa module release is updated?

Reply:
Yep!

Reply:
TODO:
- add `provision_s3_access_block` and set to false
- update mwaa source version